business_requirement_deliverable/tests/test_br.py

```diff
@@ -200,6 +200,7 @@ def test_resource_type_change(self):
         for line in self.br.deliverable_lines:
             for resource in line.resource_ids:
                 if resource and resource.resource_type == 'task':
+                    resource.resource_type_change()
```

Comment:
first method triggering for else condition then it write `resource_type=procurement` so second triggering for if condition.